angular/src/app/services/elasticsearch/elastic.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ElasticService {
+
+
+  private backendElasticsearchUrl: string = 'backend/elasticsearch';
+  constructor(private http: HttpClient) { }
+
+  getAllKeys(): Observable<any> {
+    return this.http.get<string[]>(`${this.backendElasticsearchUrl}/getAllKeys`, {});
+  }
+  getAllHostName(): Observable<any> {
+    return this.http.get<string[]>(`${this.backendElasticsearchUrl}/getAllHostName`, {});
+  }
+  getLastValue(hostName: string, itemKey: string): Observable<any> {
+    let params = new HttpParams();
+
+    params = params.append('hostname', hostName);
+    params = params.append('itemkey', itemKey);
+    return this.http.get(`${this.backendElasticsearchUrl}/getLastValue`,
+      { params: params, responseType: "text" });
```

Comment:
was wird zurück gegeben?

Reply:
responseType: "text"

Reply:
Funktioniert das in der aktuellen Version?

Reply:
ja

Reply:
so get<string> funktioniert nicht